src/VueTyping.spec.js

```diff
@@ -1,82 +1,90 @@
-import { createLocalVue, mount } from '@vue/test-utils'
-import VueTyping from '.'
+import { createLocalVue, mount } from "@vue/test-utils";
+import VueTyping from ".";
 
-function frame () {
-    return new Promise(resolve => window.requestAnimationFrame(resolve))
+function frame() {
+  return new Promise(resolve => window.requestAnimationFrame(resolve));
 }
 
-async function frames (numberOfFrames = 2) {
-    let currentFrame = 0
-    while (currentFrame < numberOfFrames) {
-        await frame()
-        currentFrame++
-    }
+async function frames(numberOfFrames = 2) {
+  let currentFrame = 0;
+  while (currentFrame < numberOfFrames) {
+    await frame();
+    currentFrame++;
+  }
 }
 
-describe('Basic rendering', () => {
-    let Vue, wrapper
-    beforeEach(() => {
-        Vue = createLocalVue()
-        wrapper = mount(VueTyping, {
-            propsData: {
-                text: 'Hello World!'
-            },
-            localVue: Vue
-        })
-    })
-
-    it('renders initial content', () => {
-        expect(wrapper.html()).toBe('<span>Hello World!</span>')
-    })
-    
-    it('Types one character per frame', async () => {
-        const text = 'Hey Guys, sup???'
-        wrapper.setProps({ text })
-        let currentText = ''
-        for (const char of text) {
-            await frame()
-            currentText += char
-            expect(wrapper.element.textContent).toBe(currentText)
+describe("Basic rendering", () => {
+  let Vue, wrapper;
+  beforeEach(() => {
+    Vue = createLocalVue();
+    wrapper = mount(VueTyping, {
+      propsData: {
+        text: "Hello World!",
+        cursorAnimationType: {
+          blinking: false,
+          cursor: "|",
+          color: "red",
+          framerate: 24
```

Comment:
I think you should add another test for the blinking cursor and disable it in the basic rendering one

Reply:
Alright, i will create a another test for the blinking cursor